src/javascript/gameInstance.js

```diff
@@ -129,18 +137,21 @@ export default class GameInstance {
     document.addEventListener(
       "keydown",
       (event) => {
-        this.onKeyDown(event.key);
+        this.onKeyDown(event.key.toLowerCase());
+        const KeyCode = {
+          KEY_LEFT: 37,
+          KEY_RIGHT: 39,
+          ESC: 27,
+          P: 80,
+        };
```

Comment:
You can pull out this `KeyCode` dict to some separate file. Right now, every time someone calls `registerInput()`, a new `KeyCode` object has to be allocated on the heap.

Reply:
Sure, for now I'll move it to file scope (maybe?)